pcsx2/GS/Renderers/Common/GSRenderer.cpp

```diff
@@ -220,7 +220,7 @@ bool GSRenderer::Merge(int field)
 			off.x = tex[i]->GetScale().x * frame_diff.x;
 		}
 
-		if (display_diff.y >= 4) // Shouldn't this be >= 2?
+		if (display_diff.y != 0) // Shouldn't this be >= 2?
```

Comment:
we want to leave this as >= 4 when scanmsk isn't on else a ton of people are gonna freak out about their games being blurry. so maybe have a parameter which allows the >= 4 when scanmsk hasn't been set to 2 or 3, but if it does get set, switch the boolean (so it forces the merge circuit to act on any offset) and it only reverts on GSReset